arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -1325,6 +1328,160 @@
         }
       }
     },
+    "RoutingProperties": {
+      "description": "The routing related properties of the IoT hub. See: https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-messaging",
+      "type": "object",
+      "properties": {
```

Comment:
nope its optional. I will add more description on whats expected when fallbackroute is not specified.